torch/onnx/__init__.py

```diff
@@ -355,6 +384,7 @@ def forward(self, x):
             input_names=input_names,
             output_names=output_names,
             opset_version=opset_version,
+            additional_decomp=_CUSTOM_ONNX_DECOMPS,
```

Comment:
Is it useful to use both _CUSTOM_ONNX_DECOMPS as well as the passed additional_decomp? In general, people may reuse other libraries, which might have registered decomposition, while they choose to pass additional_decomp? Just wondering.